contrib_bots/lib/DefineBot.py

```diff
@@ -0,0 +1,79 @@
+# See readme.md for instructions on running this code.
+import json
+import requests
+import logging
```

Comment:
Conventionally there is an expected grouping of Python imports: standard library imports, related third party imports and local application/library specific imports:

https://www.python.org/dev/peps/pep-0008/#imports

Also having them in alphabetical order will look good too.